packages/eb-deep-merge/package-lock.json

```diff
@@ -0,0 +1,4878 @@
+{
+  "name": "@empathybroker/deep-merge",
```

Comment:
Do a `npm install` from the root of the monorepo so this package-lock is updated.

Reply:
Sorry my fault.